src/sentry/integrations/slack/service.py

```diff
@@ -266,12 +259,6 @@ def _handle_parent_notification(
         json_blocks = orjson.dumps(payload.get("blocks")).decode()
         payload["blocks"] = json_blocks
 
-        extra = {
-            "channel": channel_id,
-            "thread_ts": parent_notification.message_identifier,
-            "rule_action_uuid": parent_notification.rule_action_uuid,
-        }
```

Comment:
should any of these be included in lifecycle?

Reply:
i add later in the diff

Reply:
oops my b